widgets/common/src/filters/inputs/LabelValueInput.tsx

```diff
@@ -0,0 +1,43 @@
+import type { FunctionComponent } from 'react';
+import { useRef } from 'react';
+
+import { LabelsFilterRuleOperators } from '../types';
+import LabelKeyDropdown from '../../labels/KeyDropdown';
+import LabelValueDropdown from '../../labels/ValueDropdown';
+import { CommonAttributeValueInputProps } from './types';
+
+export interface LabelValueInputProps extends Omit<CommonAttributeValueInputProps, 'onChange' | 'value'> {
+    labelKey: string;
+    labelValue: string[];
+    onKeyChange: (key: string) => void;
+    onValueChange: (value: string[]) => void;
+}
+const LabelValueInput: FunctionComponent<LabelValueInputProps> = ({
+    onKeyChange,
+    onValueChange,
+    operator,
+    toolbox,
+    labelKey,
+    labelValue
+}) => {
+    const keyDropdownRef = useRef<HTMLElement>();
+
+    return (
+        // TODO(RD-2012): Adapt label key and value dropdowns to disallow free text
+        // TODO(RD-2007): Add better styling
+        <>
+            <LabelKeyDropdown innerRef={keyDropdownRef} onChange={onKeyChange} toolbox={toolbox} value={labelKey} />
+            {(operator === LabelsFilterRuleOperators.AnyOf || operator === LabelsFilterRuleOperators.NotAnyOf) && (
```

Comment:
Optional:

A useful pattern for such matches would be:

```suggestion
            {operatorsWithValues.includes(operator) && (
```

and having an `operatorsWithValues` array that contains those 2 operators :slightly_smiling_face: 

Both approaches are fine, I'm leaving that for you to consider :smile:

Reply:
I like that idea.
Please correct me if I'm wrong, but I see I had to define `operatorsWithValues` the following way:
```
const operatorsWithValues: FilterRuleOperator[] = [LabelsFilterRuleOperators.AnyOf, LabelsFilterRuleOperators.NotAnyOf];
```
to be able to use it in
```
operatorsWithValues.includes(operator)
```

I assume that's the right way or there is simpler way to do it?

Reply:
If TS is not happy with omitting the type annotation of `FilterRuleOperator[]`, then yes, that should be the way :slightly_smiling_face:

Reply:
Yes, TS was complaining about it. OK, done.